test_tools/log_parser/test_gen.ipynb

```diff
@@ -0,0 +1,77 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "source": [
+    "from log_gen import *\n",
+    "from pathlib import Path\n",
+    "\n",
+    "root = Path('../../external_parser/unit_tests/test_files/')\n",
+    "valid_joined_logs = root.joinpath('valid_joined_logs')"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Generate pdrop test data"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "source": [
+    "ctx1 = \"\"\"{\"_multi\":[{\"a1\":\"f1\"},{\"a2\":\"f2\"}]}\"\"\"\n",
```

Comment:
I guess we can't expect that the CLI vocabulary be rich enough to produce the sort of test files we need without making it a DSL. This approach makes more sense.
I don't understand  why did you pick a notebook, they are pretty hard to review compared to regular python files and in this case we don't care for any interactive elements, do we?

Reply:
do you suggest just .py script?

Reply:
yeah I agree. replaced with python file - for now just naively converted from notebook, but I hope we can think later how exactly it should be written.